bitrise.yml

```diff
@@ -0,0 +1,127 @@
+format_version: 4
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test:
+    after_run:
+      - _test_checkout_commit
+      - _test_checkout_tag
+      - _test_checkout_branch
+      - _test_checkout_different_dir
+      - _test_ssh_git_prefix
+    steps:
+      - go-list:
+      - golint:
+      - errcheck:
+      - go-test:
+      - activate-ssh-key:
+          run_if: true
+  ci:
+    before_run:
+      - test
+  _test_checkout_commit:
```

Comment:
We separate workflows with a new line, this makes reading the bitrise.yml easier